java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -32,6 +32,7 @@
 import com.microsoft.hsg.android.simplexml.things.types.types.PersonInfo;
 import com.microsoft.hsg.android.simplexml.things.types.types.Record;
 
+
 public class MainActivity
```

Comment:
We don't need new lines  for extends and implements (unless the line length exceeds our max), in which case use minimal wrapping